src/main/java/com/bookshop/controller/BookController.java

```diff
@@ -29,7 +34,18 @@ public BookDto getBookById(@PathVariable Long id) {
     }
 
     @PostMapping
-    public BookDto createBook(@RequestBody CreateBookRequestDto requestDto) {
+    @ResponseStatus(HttpStatus.CREATED)
+    public BookDto createBook(@RequestBody @Valid CreateBookRequestDto requestDto) {
         return bookService.save(requestDto);
     }
+
+    @DeleteMapping("/{id}")
+    public void deleteById(@PathVariable Long id) {
+        bookService.deleteById(id);
+    }

```

Comment:
you also can add @Validated annotation on your controller class in order to make validation of @PathVariable, @RequestParam etc. (optional)

Reply:
pls address the comment left by @liliia-ponomarenko